spec/unit/libraries/resource_provider_spec.rb

```diff
@@ -93,6 +93,54 @@
       expect(r.item.class).to eq(OneviewSDK::API600::Synergy::Scope)
     end
 
+    it 'builds valid API800::C7000 resources' do
```

Comment:
Why do we need to add these here?

Reply:
it is require to test whether a valid resource is created as per its API version and variant class. Its not necessary to include it but i have added just to keep this file in sync with the api's we are supporting.